packages/tx/README.md

```diff
@@ -359,7 +389,7 @@ const eth = new Eth(transport)
 const common = new Common({ chain: Chain.Sepolia })
 
 let txData: any = { value: 1 }
-let tx: Transaction | FeeMarketEIP1559Transaction
+let tx: LegacyTransaction | FeeMarketEIP1559Transaction
```

Comment:
Should this be moved into an example file as well?

Reply:
Ideally, yes, but in order to make it work we need to install that ledger dependency.  Let me add it as a dev dependency and see if it even works (given that the whole idea of this example would require someone to have a ledger handy)

Reply:
Okay, so digging into this further means a ton of work to make this example work out of the box. Having looked at the code in the Ledger dep, I think the example is sound as is.  I'd need to implement an entire fake transport class to mimic a Ledger device in order for this to work "as-is".  That might be worth it, but it's probably beyond the scope of this PR.

Reply:
Ok, agreed :+1: